plugin/src/PatchTree.lua

```diff
@@ -83,6 +86,8 @@ function Tree:forEach(callback, node, depth)
 	end
 end
 
+-- Finds a node by id, depth first
+-- target is the node to search within, defaults to root
 function Tree:getNode(id, target)
```

Comment:
If `target` is a clearer name like `nodeToSearchIn` it wouldn't need a comment saying it's the node to search in.